Index: src/Basic/temperature.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Basic/temperature.java b/src/Basic/temperature.java
new file mode 100644
--- /dev/null	(date 1681552993518)
+++ b/src/Basic/temperature.java	(date 1681552993518)
@@ -0,0 +1,16 @@
+package Basic;
+
+import java.util.Scanner;
+
+public class temperature {
+    public static void main(String []args){
+        Scanner sc=new Scanner(System.in);
+        System.out.print("Fahrenheit : ");
+        float Fahrenheit = sc.nextFloat();
+        System.out.println("Temperature in Celsius : " +TemperatureConv(Fahrenheit));
+    }
+    public static double TemperatureConv(float Fahrenheit){
+        return ((Fahrenheit-32)/1.8);
+
+    }
+}
Index: src/Basic/Simple_interest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Basic/Simple_interest.java b/src/Basic/Simple_interest.java
new file mode 100644
--- /dev/null	(date 1681553489365)
+++ b/src/Basic/Simple_interest.java	(date 1681553489365)
@@ -0,0 +1,18 @@
+package Basic;
+import java.util.Scanner;
+public class Simple_interest {
+    public static void main(String []args){
+        Scanner sc= new Scanner(System.in);
+        int PI, time, rate,SI;
+        System.out.print("Principle amount : ");
+         PI = sc.nextInt();
+         System.out.print("Time : ");
+         time= sc.nextInt();
+         System.out.print("rate of  interest : ");
+         rate= sc.nextInt();
+         SI = ((PI * rate * time)/100);
+         System.out.println("Simple interest : "+SI);
+
+
+    }
+}
Index: src/Basic/marks_percentage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Basic/marks_percentage.java b/src/Basic/marks_percentage.java
new file mode 100644
--- /dev/null	(date 1681551927256)
+++ b/src/Basic/marks_percentage.java	(date 1681551927256)
@@ -0,0 +1,23 @@
+package Basic;
+
+import java.util.Scanner;
+
+public class marks_percentage {
+    double total , percentage;
+    public static void main(String []args){
+        Scanner sc=new Scanner(System.in);
+        System.out.print("Maths : ");
+        float Maths = sc.nextFloat();
+        System.out.print("Computer : ");
+        float Computer = sc.nextFloat();
+        System.out.print("English : ");
+        float English =sc.nextFloat();
+      double  total = Maths + Computer + English;
+        System.out.println("Total marks: "+total);
+
+      double  percentage =((total/300.0)*100);
+        System.out.print("percentage : "+percentage);
+
+
+    }
+}
