Index: src/Basic/fifty_two.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Basic/fifty_two.java b/src/Basic/fifty_two.java
new file mode 100644
--- /dev/null	(date 1682097468406)
+++ b/src/Basic/fifty_two.java	(date 1682097468406)
@@ -0,0 +1,4 @@
+package Basic;
+
+public class fifty_two {
+}
Index: src/Basic/fifty_one.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Basic/fifty_one.java b/src/Basic/fifty_one.java
new file mode 100644
--- /dev/null	(date 1682097468415)
+++ b/src/Basic/fifty_one.java	(date 1682097468415)
@@ -0,0 +1,4 @@
+package Basic;
+
+public class fifty_one {
+}
